src/content/guides/creating-clusters-via-crs-on-azure/index.md

```diff
@@ -189,3 +189,27 @@ There are also specific reference pages for [cluster templating](/reference/kube
 As a result of rendering the CRs ([sample](/reference/kubectl-gs/template-cluster/#example)), a user will get YAML manifests containing valid CRs that can create a tenant cluster and its node pools.
 The resources can then be created by applying the manifest files to the Control Plane, e.g. `kubectl create -f <cluster manifest file>.yaml`.
 Of course, that requires the user to be authorized towards Kubernetes Control Plane API.
+
+## How to configure OIDC authentication at cluster creation using Cluster API
+
+Starting from version {{% first_azure_nodepools_version %}} on Azure we have enabled the possibility to configure the OIDC per cluster at the cluster creation stage. This feature only includes configuration of the OIDC in the `Cluster` CR before it is applied on the Control Plane.
+
+In order to configure your new cluster with OIDC, you will have to add adjusted to your needs annotations to the Cluster CR:
```

Comment:
I don't get the meaning of this sentence

Reply:
simplified